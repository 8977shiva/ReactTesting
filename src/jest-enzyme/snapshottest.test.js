import React from "react";
import renderer from "react-test-renderer";
import Todo from "../todo";

it("creating snapshot", () => {
  const comp = renderer.create(<Todo />);
  let tree = comp.toJSON();
  expect(tree).toMatchSnapshot();
});
