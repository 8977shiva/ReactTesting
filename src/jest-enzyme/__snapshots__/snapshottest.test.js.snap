// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creating snapshot 1`] = `
<div>
  <h3>
    TODO
  </h3>
  <form
    onSubmit={[Function]}
  >
    <label
      htmlFor="new-todo"
    >
      What needs to be done?
    </label>
    <input
      id="new-todo"
      onChange={[Function]}
      value=""
    />
    <button>
      Add #
      1
    </button>
  </form>
  <div>
    <ul />
  </div>
</div>
`;
